docs/api/sensors.md

```diff
@@ -26,7 +26,7 @@ Trends, etc., during the COVID-19 pandemic, because these were designed to track
 ILI as driven by seasonal influenza, and were NOT designed to track ILI during
 the COVID-19 pandemic.
 
-We are working on collecting new ILI-related data sources may be useful to
+We are working on collecting new ILI-related endpoints may be useful to
 nowcasting and forecasting ILI during the COVID-19 pandemic, and will make these
 available as soon as possible.
```

Comment:
@RoniRos is this paragraph relevant anymore? if not we should delete it

```suggestion
```

Reply:
No longer relevant, and should be deleted.  (Also "Data sources" was not used to mean 'endpoints'.)

Also, @krivard : I noticed both above and below numerous (in fact, most!) cases where 'data source' was replaced by 'endpoint' even though that was not the intended meaning of that term - it should be left as 'data source'.   This substitution should not be done automatically!

Also:

> This is the documentation of the API for accessing the Delphi's Digital Surveillance Sensors (`sensors`) endpoint of
the [Delphi](https://delphi.cmu.edu/)'s epidemiological data. 

This is garbled and overly wordy.  I suspect it should be something like:

"This is the documentation of the API for accessing Delphi's Epidata -- epidemiological sensors and indicators database. "